examples/pytorch/model_zoo/chem/property_prediction/README.md

```diff
@@ -41,8 +41,59 @@ match exact results for this model, please use the pre-trained model as in the u
 To customize your own dataset, see the instructions
```

Comment:
Let's move the dataset configuration part to be after the whole regression section.